Sources/Actions/Moderation/ReportedContent.php

```diff
@@ -1215,6 +1215,10 @@ protected function saveModComment(int $report_id, array $data): void
 
 		$report = $this->getReportDetails($report_id);
 
+		if (empty($report)) {
+			return;
+		}
+
```

Comment:
While we're at it, we should probably move all of this chunk to the top of the function, before we even try to insert the comment. After all, if the report doesn't exist, what are we commenting on?